### DISCUSSION TITLE
Use Material 3 switch

### DISCUSSION THREAD

**Discussion by Lijucay** [Droid-ify/client#251] [Ideas]
Using Google's Material 3 is very cool. However, I realised that the switches you are using are not really what Google introduced in their M3 release.
My idea was to change the switches to what the following picture shows:

![unnamed.png](https://user-images.githubusercontent.com/104652374/218270957-e0f56c11-336b-4b86-b933-ab9cc0d68f24.png)

I can add a link to Google's M3 - switch component overview:

https://m3.material.io/components/switch/overview

**Comment by Iamlooker** [comment]
Done
